frontend/lib/src/components/elements/CodeBlock/StreamlitSyntaxHighlighter.test.tsx

```diff
@@ -73,4 +73,11 @@ describe("CustomCodeTag Element", () => {
       baseElement.querySelector("pre code .token.string")?.innerHTML
     ).toBe('"Hello"')
   })
+
+  it("applies height style when height prop is provided", () => {
```

Comment:
Not sure if we actually need that unit test given that a) we basically test the same thing in the e2e test and b) we don't have unit tests for any of the other parameters in `st.code`.